lib/features/onboarding/QA.md

```
@@ -0,0 +1,128 @@
+</details>
+
+</details>
+
+---
+
+## 3. Analytics
+
+- [x] **ファネルと購入のAnalyticsイベント**: 開始・各画面・回答・完了・1画面目のシステム戻るで閉じた操作を記録し、実際のentitlementがtrialならtrial_start、paidならpurchase_completeをstoreProductIdentifier付きで記録する
+  - 自動化: auto（test/features/onboarding/onboarding_page_test.dart、test/features/paywall/paywall_page_test.dart）
+
+#### 動作確認
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)